tests/framework/CodeInterfaceTests/Serpent/SerpentTest1/sampleInput/2/testInput.serpent.seed

```diff
@@ -0,0 +1 @@
+1523667329
```

Comment:
This file is read by SERPENT iteself. It is not needed for testing the interface. Please remove it